dist/less/ds6/mixins.less

```diff
@@ -106,7 +106,7 @@
 
 // base styles for BEM svg.icon block
 // deprecated for the common mixin foreground-icon-base()
-.inline-icon-base(@width: 1rem; @height: 1rem) {
+.foreground-icon-base(@width: 1rem; @height: 1rem) {
```

Comment:
foreground-icon-base is common to ds4 and ds6, so shouldn't be showing up here. Maybe just need to run a `yarn build` again to regenerate this dist file...?

Reply:
Oh crap. That was a miss. Fixing ...

Reply:
I'm pretty sure I left inline-icon-base in here just for backwards compatibility (i.e. apps using this mixin). But as it's a major version change we can remove it if we want.